sdk/core/azure-core/tests/async_tests/test_rest_http_response_async.py

```diff
@@ -242,6 +242,17 @@ async def test_multipart_data_and_files_content(send_request):
     await send_request(request)
 
 
+@pytest.mark.asyncio
+async def test_multipart_data_and_named_files_content(send_request):
```

Comment:
THose test should not be in a 'response" file

Reply:
I think it is better to test it like other cases: https://github.com/Azure/azure-sdk-for-python/blob/ad3cf0cc892a667162d0776c8e7f610d632135d9/sdk/core/azure-core/tests/test_rest_http_response.py#L202-L218

Reply:
+1 to @lmazuel

Reply:
+1 to @iscai-msft

Reply:
Fine. 
Moved.